CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Guide lines for contribuition in our gitbook!
+* [http://dev.docs.lightblue.io/development_process/README.html](http://dev.docs.lightblue.io/development_process/README.html)
```

Comment:
Some other text would be nice here instead of the URL.  Maybe "Development Process".